lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -306,9 +314,9 @@ export default function BasicLTILaunchApp() {
 
   const content = (
     <div
-      // Visually hide the iframe / grader if there is an error or no contentUrl.
+      // Visually hide the iframe / grader if there is an error or no contentURL.
```

Comment:
The check is for `contentReady` rather than `contentURL`. Honestly though I think the code is obvious enough that we could omit this comment.

Reply:
Yeah I almost deleted it in my first pass. GOOD BYE.